src/base/scanfoldersmodel.cpp

```diff
@@ -392,3 +387,18 @@ void ScanFoldersModel::addTorrentsToSession(const QStringList &pathList)
         }
     }
 }
+
+QString ScanFoldersModel::pathTypeDisplayName(const PathType type)
+{
+    switch(type) {
+    case DOWNLOAD_IN_WATCH_FOLDER:
+        return tr("Same as monitored folder");
```

Comment:
Then it should be "The same location as monitored folder", maybe the "location" cannot be omitted or the sentence wouldn't be complete.
sigh... I'll just use "Monitored folder".